tembo-operator/src/controller.rs

```diff
@@ -309,6 +312,22 @@ impl CoreDB {
                 let (trunk_installs, extensions) =
                     reconcile_extensions(self, ctx.clone(), &coredbs, &name).await?;
 
+                // Make sure the initial backup has completed if enabled before finishing
+                // reconciliation of the CoreDB resource
+                if cfg.enable_backup
+                    && self
+                        .status
+                        .as_ref()
+                        .and_then(|s| s.first_recoverability_time)
+                        .is_none()
+                {
+                    check_backups_status(self, ctx.clone()).await?;
+                    // if let Err(e) = check_backups_status(self, ctx.clone()).await {
+                    //     error!("Error checking backups status: {:?}", e);
+                    //     return Err(Action::requeue(Duration::from_secs(300)));
+                    // }
+                }
+
```

Comment:
Not sure if this is the best place, but ideally it needs to be the last thing ran before we wrap up reconciliation.  We also might want to do this on restores only and not initial instances.